src/main/java/io/cdap/plugin/google/sheets/sink/GoogleSheetsSinkConfig.java

```
@@ -167,7 +167,7 @@ public class GoogleSheetsSinkConfig extends GoogleInputSchemaFieldsUsageConfig {
    * @param schema the schema to check compatibility
    */
   public void validate(FailureCollector collector, Schema schema) {
-    super.validate(collector);
+    super.getValidationResult(collector);
```

should we do `collector.getOrThrowException` after this call everywhere?

---

is this required ? `getOrThrowException` is already called from configurePipeline/prepareRun at the end of all validations !

---

This will allow us to fail early, if the initial validation fails, there is no point of going forward

---

Added !